react-native-gutenberg-bridge/android/build.gradle

```diff
@@ -117,6 +117,9 @@ dependencies {
     implementation "org.webkit:android-jsc:r241213"
     implementation project(':react-native-aztec')
 
+    // For animated GIF support
+    implementation 'com.facebook.fresco:animated-gif:2.0.0'
```

Comment:
Good question @cameronvoell!
I was thinking to remove it but then I realize that we might need it as the user could already have `gif` in his Wordpress Library and it would be cool that he can see animated `gif` while editing the Post.

Reply:
Just tested this, very cool!